src/main.cairo

```diff
@@ -88,48 +144,37 @@ mod QuestNft {
         user_addr: felt252,
     }
 
-    mod Errors {
-        const INVALID_TOKEN_ID: felt252 = 'ERC721: invalid token ID';
-        const INVALID_ACCOUNT: felt252 = 'ERC721: invalid account';
-        const UNAUTHORIZED: felt252 = 'ERC721: unauthorized caller';
-        const APPROVAL_TO_OWNER: felt252 = 'ERC721: approval to owner';
-        const SELF_APPROVAL: felt252 = 'ERC721: self approval';
-        const INVALID_RECEIVER: felt252 = 'ERC721: invalid receiver';
-        const ALREADY_MINTED: felt252 = 'ERC721: token already minted';
-        const WRONG_SENDER: felt252 = 'ERC721: wrong sender';
-        const SAFE_MINT_FAILED: felt252 = 'ERC721: safe mint failed';
-        const SAFE_TRANSFER_FAILED: felt252 = 'ERC721: safe transfer failed';
+    #[derive(Drop, starknet::Event)]
+    struct on_mint {
```

Comment:
check that the name keys and values are the same than before so that indexing works